.github/workflows/build_and_test.yml

```
@@ -102,8 +102,22 @@ jobs:
              export SDE_INSTALL=${GITHUB_WORKSPACE}/install
              export DEPEND_INSTALL=${GITHUB_WORKSPACE}/deps
              source ./scripts/dpdk/setup_env.sh $IPDK_RECIPE $SDE_INSTALL $DEPEND_INSTALL
-             ./make-all.sh --target=dpdk  --prefix=${GITHUB_WORKSPACE}/dpdk-target 
-          
+             ./make-all.sh --target=dpdk  --prefix=${GITHUB_WORKSPACE}/dpdk-target
+
+      - name: 'Run Krnlmon Unit Test for DPDK Target'
+        run: |
+             cd ${GITHUB_WORKSPACE}/ipdk.recipe
+             cmake -B build -DDPDK_TARGET=on -DWITH_OVSP4RT=off
+             cmake --build build -j4 --target krnlmon-test
+             more build/Testing/Temporary/LastTest.log
```

```suggestion
             cat build/Testing/Temporary/LastTest.log
```
`more` is used in interactive environments. This is a batch environment.

---

fixed!